src/main/java/pl/edu/pw/pik/pikactivitytrackerserver/controller/UserController.java

```diff
@@ -31,4 +32,30 @@
 
         return new ResponseEntity<>(null, HttpStatus.NOT_ACCEPTABLE);
     }
+
+    @GetMapping("/login/{username}")
+    public ResponseEntity<String> loginUsername(@PathVariable String username)
+    {
+        String salt = userService.loginUsername(username);
+
+        if(salt != null)
+        {
+            return new ResponseEntity<>(salt, HttpStatus.OK);
+        }
+
+        return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+    }
+
+    @PostMapping("/login")
+    public ResponseEntity<Integer> loginUsernameAndHashedPassword(@RequestBody UserDTO dto)
```

Comment:
Tutaj jest druga funkcja do logowania. Ich nazwy są podobne. Do końca nie wiem, kiedy miałaby być jedna użyta, a kiedy druga. Może warto to zunifikować i wypluwać jakiś nowy DTO?

Reply:
zmieniłam na login